chore: update README to add an example (#30)
oras-project/setup-oras: README.md

## Advanced Usage

Install a specific version of the `oras` CLI by specifying the input `version` without the prefix `v`.

> Stable versions (not `beta` or `rc` versions) that can be found at [`oras` releases](https://github.com/oras-project/oras/releases) are supported.

Reviewer: Suggest not to mentioned `beta` or `rc`, since there can also be `alpha`.
```suggestion
Stable versions (not pre-release versions) are supported and can be found at [`oras` releases](https://github.com/oras-project/oras/releases) are supported.
```

  - run: oras version
```

> You can also download `oras` CLI from any preferred trusted source, by specifying link of the source via `url` and the SHA256 checksum via `checksum`. This can be used to download a `beta` or `rc` version, or download from a proxy. 

Reviewer: ```suggestion
You can also download `oras` CLI from any preferred trusted source, by specifying a link of the source via `url` and the SHA256 checksum via `checksum`. This can be used to download a pre-release version or download from a proxy other than `github.com`. 
```

  - run: oras version
```

> You can also download `oras` CLI from a customized link, by specifying the inputs `url`

Reviewer: ```suggestion
You can also download `oras` CLI from any preferred trusted source, by specifying link of the source via `url`
```

> and `checksum`. This can be used to download a `beta` or `rc` version, or download 

Reviewer: ```suggestion
and the SHA256 checksum via `checksum`. This can be used to download a `beta` or `rc` version, or download 
```

> from a proxy. Both inputs must be provided. 

Reviewer: ```suggestion
from a proxy.
```
Suggest removing the constraint to keep example short.

> When both `url` and `checksum` are provided, the `version` value is ignored and the 
> download will be from `url` and checked against `checksum`.

Reviewer: Suggest removing the constraint to keep example short.
```suggestion
```
